webapp/Connector/measure.js

```diff
@@ -635,7 +635,9 @@ Ext.define('Connector.measure.Configuration', {
                     isRecommendedVariable: true
                 },
                 'study_PKMAb_mab_mix_id': {
-                    hiddenInPlot: true
+                    hiddenInPlot: true,
+                    allowForGrouping: true,
```

Comment:
The other 3 fields should also have this property set. I understand they are not strictly needed since they are included in plot data as default dimensions. But having explicit annotation makes it clear which fields are for grouping:
study_PKMAb_lab_code
study_PKMAb_source_assay
study_PKMAb_specimen_type

Reply:
Adding 'isLinePlotGroupingField' to these fields came in handy in constructing groupField, so this was a good addition overall.